packages/devtools_app/lib/src/inspector/inspector_screen.dart

```diff
@@ -296,3 +371,72 @@ class InspectorScreenBodyState extends State<InspectorScreenBody>
     });
   }
 }
+
+class ErrorNavigator extends StatelessWidget {
+  const ErrorNavigator(
+      {Key key,
+      @required this.selectedErrorIndex,
+      @required this.errorCount,
+      @required this.onSelectedErrorIndexChanged})
+      : super(key: key);
+
+  final int selectedErrorIndex;
+  final int errorCount;
+  final void Function(int) onSelectedErrorIndexChanged;
+
+  @override
+  Widget build(BuildContext context) {
+    final label = selectedErrorIndex != null
+        ? 'Error ${selectedErrorIndex + 1}/$errorCount'
+        : 'Errors: $errorCount';
+    return Container(
+      color: devtoolsError,
+      child: Padding(
+        padding: const EdgeInsets.symmetric(horizontal: 12, vertical: 6),
+        child: Row(
+          children: [
+            Padding(
+              padding: const EdgeInsets.only(right: 6),
+              child: Text(label),
+            ),
+            IconButton(
+              padding: EdgeInsets.zero,
+              constraints: const BoxConstraints(),
```

Comment:
why the empty box constraints?

Reply:
Without this, the IconButtons seem to force the container to be quite large:

![Screenshot 2021-02-08 at 18 56 38](https://user-images.githubusercontent.com/1078012/107267916-68488880-6a3f-11eb-80e8-5361d133f179.png)

I'm not certain this is the correct fix though?

Reply:
yeah that's not great either. fine to leave as is - not worth blocking commit over 👍